diff --git a/node_modules/react-native-css-interop/.cache/android.js b/node_modules/react-native-css-interop/.cache/android.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/ios.js b/node_modules/react-native-css-interop/.cache/ios.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/macos.js b/node_modules/react-native-css-interop/.cache/macos.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/native.js b/node_modules/react-native-css-interop/.cache/native.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/windows.js b/node_modules/react-native-css-interop/.cache/windows.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
index 59ea66e..c434709 100644
--- a/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
+++ b/node_modules/react-native-css-interop/dist/css-to-rn/parseDeclaration.js
@@ -1761,6 +1761,9 @@ function parseAspectRatio(aspectRatio) {
         return "auto";
     }
     else {
+        if (!aspectRatio.ratio) {
+            return undefined;
+        }
         if (aspectRatio.ratio[0] === aspectRatio.ratio[1]) {
             return 1;
         }
